### ISSUE TITLE
Prediction Issue... Need your wisdom...

### ISSUE THREAD

**Issue by bizcjh1027** [shekkizh/FCN.tensorflow#81]
Hello.

My prediction is like below... Do you have any idea to change things to be more like ground truth?

Dependencies & Parameters & images are below

Cudnn : 6.0
Cuda : 8.0
Tensorflow-gpu : 1.3
Learning Rate : 1e-5
Batch Size : 16
Max_Iteration : 25,300
Epoch : 20
Class : 151

Input
![inp_19](https://user-images.githubusercontent.com/18678655/35600086-6c1db976-066f-11e8-9e12-32931646586d.png)

Ground Truth
![gt_19](https://user-images.githubusercontent.com/18678655/35600091-6fdbc7d8-066f-11e8-9683-2b08caceb7fa.png)

Prediction
![pred_19](https://user-images.githubusercontent.com/18678655/35600092-71842cba-066f-11e8-932f-435fade1e91e.png)

**Comment by shekkizh** [comment]
more training. class balance. need more info to comment. 
Closing due to inactivity
